package/gluon-mesh-batman-adv-brmldproxy/files/usr/sbin/gluon-brmldproxy-router-check

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+BRMLDPROXY_RTR_GROUP="ff12::424e:f314"
```

Comment:
Maybe add a comment regarding the source of this group address?

Using a group prefix derived from `prefix6` (as specified in some RFC) might also be a an alternative.